core/shared/src/main/scala/matryoshka/package.scala

```diff
@@ -35,101 +35,103 @@ import scalaz._, Scalaz._
   * @groupdesc dist Natural transformations required for generalized folds and unfolds.
   */
 package object matryoshka {
+  type Fix[F[_]] = matryoshka.data.Fix[F]
+  val Fix        = matryoshka.data.Fix
```

Comment:
@japgolly It’s just that there are type classes for fixed-point types. `Fix` is one specific type, and in general, algebras, folds, etc. can be defined generically – using something like `T[_[_]]: Recursive` (there exists `Recursive[Fix]`, so no worries). Then, you can instantiate it later using `Fix` … or `Mu`, or `Nu`.

So, define everything generically, then specify the type only at the boundaries.